types/src/crypto/asymmetric_key.rs

```diff
@@ -84,19 +85,12 @@ where
             return Err(Error::AsymmetricKey("too short".to_string()));
         }
 
-        let (tag_bytes, key_bytes) = input.as_ref().split_at(2);
-        let mut tag = [0u8; 1];
-        hex::decode_to_slice(tag_bytes, tag.as_mut())?;
+        let bytes = checksummed_hex::decode(&input)?;
+        let (tag, key_bytes) = bytes.split_at(1);
```

Comment:
Why do we split at `(1)` now where previously it was on `(2)`? `bytes` is a `[u8]` still, right? Does that mean that the key tag is now only 1 byte instead of 2?

Reply:
@xcthulhu, can you answer this?

Reply:
It's pretty much equivalent.  We used to split before decoding, hence taking two hex chars (as `u8`s) to deduce the tag by decoding just those two hex chars to a byte.  Now we decode from hex to bytes first, then take the first byte.